### PR TITLE
Feat(STN-835): spotlight slider styles

### PR DESCRIPTION
**Pull request by pebutler3**
# READY FOR REVIEW

## Summary
Utilized slider mixins to implement styles for Spotlight Slider. 
No new styles were added.

## Need Review By (Date)
asap

## Designs
[figma](https://www.figma.com/file/BieDQhj8BoLlHJrzpDaIXV/Slider-Container-for-Hero-Components?node-id=0%3A1)

## Urgency
medium

## Steps to Test
1. Create a new Content Page and utilizing the newly added Spotlight Slider and confirm designs match figma (both full-width and inside Well)
2. Add Gradient Hero Slider component and confirm its styles haven't changed.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

**Review by rkelbel — COMMENTED** [review]
This looks great overall, one thing I did notice is that Spotlight can contain different amounts of text and links, which makes the size of each slide different.  This causes the slider to shift is the size when swiping between.  Is this something H&S is expecting as behavior or do they want us to do something to prevent this shifting?

screencap

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr973-hyescsjk5byboocbvsh2wamcapc9eb1t.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156533a4807c04df53ff173

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr973-hyescsjk5byboocbvsh2wamcapc9eb1t.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156533a4807c04df53ff173

**Comment by pebutler3** [comment]
@rkelbel `carousel-slides-height.js` has been updated to ensure equal height is applied to Spotlight Sliders regardless of height. I suspect that in the future we could remove the conditional logic in there to have it applied to all carousel functionality.

File: docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.spotlight.scss [hunk 1]
```diff
@@ -111,3 +111,19 @@
...
+// Spotlight Slider styles
+.paragraph--type--hs-sptlght-slder {
+  @include hb-hero-full-width();
+  @include slick-container;
+  @include slick-dots;
+  @include slick-arrows;
+  position: relative;
+
+  .hs-full-width & {
+
```
Comment (rkelbel) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.spotlight.scss]: Line 124 is causing an unexpected empty line linting error.

Comment (rkelbel) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.spotlight.scss]: Fixed 👍🏻

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr973-hyescsjk5byboocbvsh2wamcapc9eb1t.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156533a4807c04df53ff173

**Review by rkelbel — APPROVED** [review]
Looks great!

thumbs up

**Pull request merged**